packages/commercetools/composables/src/getters/orderGetters.ts

```diff
@@ -1,26 +1,69 @@
 import { UserOrderGetters, AgnosticOrderStatus } from '@vue-storefront/interfaces';
-import { Order, OrderState } from './../types/GraphQL';
+import { Order, OrderState, LineItem, Money, Address } from './../types/GraphQL';
 
 export const getOrderDate = (order: Order): string => order?.createdAt || '';
 
 export const getOrderId = (order: Order): string => order?.id || '';
 
+export const getOrderNumber = (order: Order): string => order?.orderNumber || getOrderId(order);
```

Comment:
Should such fallback be hidden like this under the getter?

Reply:
I think it should. If the API doesn't create an order number, then we should use the ID of the entity. It's more agnostic I think